test/integration/metrics/metrics_test.go

```diff
@@ -105,9 +105,6 @@ func TestAPIServerStorageMetrics(t *testing.T) {
 }
 
 func TestAPIServerMetrics(t *testing.T) {
-	// KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE allows for APIs pending removal to not block tests
-	t.Setenv("KUBE_APISERVER_SERVE_REMOVED_APIS_FOR_ONE_RELEASE", "true")
-
 	s := kubeapiservertesting.StartTestServerOrDie(t, nil, framework.DefaultTestServerFlags(), framework.SharedEtcd())
```

Comment:
I would lock the binary version to 1.33 to prevent future failures 
```
&kubeapiservertesting.TestServerInstanceOptions{BinaryVersion: "1.32"}
```

It would work until AdmissionregistrationV1beta1 file is removed from the code base. Meanwhile we would have more time to work on a long term solution with fake api.

Reply:
Fixed